dashboard/src/utils/checkStatus.js

```diff
@@ -0,0 +1,29 @@
+import { compose } from "ramda";
```

Comment:
I think this should be implemented in a `/src/config/checkStatus.js` file exported as flat object maps.